experimental/unitsconversion/src/provider.rs

```diff
@@ -33,5 +33,65 @@ pub struct UnitsConstantsV1<'data> {
     // Also, the constant types.
     /// Maps from constant name (e.g. ft_to_m) to the value of the constant (e.g. 0.3048).
     #[cfg_attr(feature = "serde", serde(borrow))]
-    pub constants_map: ZeroMap<'data, str, str>,
+    pub constants_map: ZeroMap<'data, str, ConstantValueULE>,
+}
+
+/// This enum is used to represent the type of a constant value.
+/// It can be either `ConstantType::Actual` or `ConstantType::Approximate`.
+/// If the constant type is `ConstantType::Approximate`, it indicates that the value is not definitively accurate.
+#[zerovec::make_ule(ConstantTypeULE)]
+#[cfg_attr(
+    feature = "datagen",
+    derive(serde::Serialize, databake::Bake),
+    databake(path = icu_unitsconversion::provider),
+)]
+#[cfg_attr(feature = "serde", derive(serde::Deserialize))]
+#[derive(Copy, Clone, Debug, PartialOrd, Ord, PartialEq, Eq, Default)]
+#[repr(u8)]
+pub enum ConstantType {
```

Comment:
Nit: In general, avoid enums named `Type` or `Kind` if they describe only one aspect of the object. In this case, `ConstantType` should probably be `ConstantExactness` or something.

Reply:
https://github.com/unicode-org/icu4x/issues/4092

Reply:
#4092 is not this issue